api/tests/unit/organisations/subscriptions/serializers/test_unit_subscriptions_serializers_mixins.py

```diff
@@ -8,20 +9,29 @@
 )
 
 
-def test_read_only_if_not_valid_plan_mixin_sets_read_only_if_plan_not_valid():
+@pytest.mark.parametrize(
+    "plan_id, invalid_plans_, invalid_plans_regex_",
+    (
+        ("invalid-plan-id", ("invalid-plan-id",), ""),
+        ("invalid-plan-id", tuple(), "invalid-.*"),
+    ),
+)
```

Comment:
Do we want a test for valid plans? I guess it is covered by other existing tests?

Reply:
Yeah, that is handled [here](https://github.com/Flagsmith/flagsmith/pull/3607/files#diff-ab9484110ccbb67445f721445b99c61988f49a0f13f44ef34110efd7c48c7d5fR46), although I guess it doesn't technically test with the regex matcher. I'll add that quickly.